internal/functions/serve/templates/main.ts

```diff
@@ -187,11 +187,6 @@ Deno.serve({
     const cpuTimeSoftLimitMs = 1000;
     const cpuTimeHardLimitMs = 2000;
 
-    // NOTE(Nyannyacha): Decorator type has been set to tc39 by Lakshan's request,
-    // but in my opinion, we should probably expose this to customers at some
-    // point, as their migration process will not be easy.
-    const decoratorType = "tc39";
```

Comment:
Feel like we should leave it in as both deno 1 and 2 share the same main.ts file for serving locally.

cc @nyannyacha wdyt?